docs/zotero.html

```diff
@@ -229,14 +229,18 @@ <h2>Zotero Library Shout-Outs</h2>
         </main>
         <script>
             // Pulse Animation for Network SVG
+
             const simulateBtn = document.getElementById('simulateBtn');
             const pulse = document.getElementById('pulse');
+
+
             
-            simulateBtn.addEventListener('click', () = > {
+            simulateBtn.addEventListener('click', () => {
+
                 pulse.setAttribute('r', '24');
                 pulse.setAttribute('opacity', '0.8');
                 let radius = 24;
-                const interval = setInterval(() = > {
+                const interval = setInterval(() => {
```

Comment:
@dal5842 Basically I changed all these to the proper JS arrow operator... That fixed it so the JS should work on the SVG pulse effect.